app/src/components/UI/Pages/CreateVendorPage.tsx

```diff
@@ -24,6 +25,7 @@ const CreateVendorPage: React.FC = () => {
   return (
     <Container className={styles.wrapper}>
       <HeaderBar logout />
+      <MessageError />
```

Comment:
Could you remove this for just the PR? You can keep the MessageError.tsx file.

Reply:
ok